packages/client/modules/meeting/components/ReflectTemplateItem.tsx

```diff
@@ -62,7 +63,8 @@ const ReflectTemplateItem = (props: Props) => {
   const description = makeTemplateDescription(lowestScope, template)
   const atmosphere = useAtmosphere()
   const ref = useRef<HTMLLIElement>(null)
-  useScrollIntoView(ref, isActive)
+  const isInit = useInitialRender()
+  useScrollIntoView(ref, isInit)
```

Comment:
Previously, if you click on any item in the menu, it would scroll up which was kinda annoying. I would lose my position on the menu. Instead, I'm now scrolling to the active item when the page loads, but not scrolling when the user clicks other items.